spec/index.bs

```diff
@@ -5296,14 +5299,39 @@ dictionary GPUExternalTextureDescriptor
 
                     Otherwise:
 
-                    1. If |source| <l spec=html>[=is not origin-clean=]</l>,
-                        throw a {{SecurityError}} and return.
+                    1. If |source| is a {{GPUTextureView}}:
+
+                        1. If any of the following conditions are unsatisfied
```

Comment:
Could this maybe be on the device timeline and produce an expired / error external texture? That's one point of detail that will need further discussion I think.